libs/islamic-website/ui-kit/src/lib/member-card/member-card.tsx

```diff
@@ -59,12 +61,12 @@ export function MemberCard({
                 event.stopPropagation();
               }}
               className={clsx(
-                'text-islamic-primary-green hover:text-islamic-primary-green-hover font-mono uppercase',
+                'text-islamic-primary-green hover:text-islamic-primary-green-hover rtl:font-handjet font-mono  uppercase',
```

Comment:
remove double spaces everywhere

```suggestion
                'text-islamic-primary-green hover:text-islamic-primary-green-hover rtl:font-handjet font-mono uppercase',
```

Reply:
Done